### ISSUE TITLE
Compress xml and json files too

### ISSUE THREAD

**Issue by Buzut** [weyusi/hexo-yam#2]
Awesome application!
I'm using it in prod right now and it works perfectly.

Nontheless, I have an atom.xml and api.json (respectively generated with `hexo-generator-feed` and `hexo-generator-json-content`) that would very much profit to be compressed.

It would be great to either directly compress them or add an option to be able to compress custom files. All the more that the api.json is pretty heavy.

**Comment by weyusi** [comment]
currently I don't have xml and json files to play with. I added support of those files in a separate branch, can you test it by modifying hexo-yam line of your `package.json` to:
```
    "snyk": "git://github.com/weyusi/hexo-yam.git#xml-json"
```
let me know how it goes.

**Comment by Buzut** [comment]
@weyusi thank you very much for your reactivity (and excuse me for my lack of one…).
It indeed works perfectly (both gzip & brotli). I'm glad you could update this.

I'm going to write down a blog article on my recent migration to Hexo and won't forget to mention your awesome plugin, it deserves more attention.

**Comment by weyusi** [comment]
implemented in v1.1.0. Thanks for your suggestion.

**Comment by Buzut** [comment]
I've been using it happily since then :)
While I'm at it, I've noticed another file type that could benefit from compression: svg. 
I don't know if that's possible but if yes, it would be great!

**Comment by weyusi** [comment]
Good suggestion. I used [svgo](https://github.com/svg/svgo) before in an unrelated project. The [reference code](https://github.com/svg/svgo/blob/master/examples/test.js) needs some modifications to fit in hexo workflow. I'll see what I can do. PR welcome.

**Comment by weyusi** [comment]
@Buzut svgo minify has been implemented in [v2.0.0](https://github.com/weyusi/hexo-yam/releases/tag/v2.0.0).

**Comment by Buzut** [comment]
Awesome! Thank you so much 💯
